src/_locales/ru/messages.json

```diff
@@ -169,26 +169,26 @@
     "html_suspended_donation_question": { "message": "Вам понравилось это расширение?\nОтправьте пожертвование." },
     "html_permissions_title": { "message": "Разрешения" },
     "html_permissions_heading": { "message": "Пожалуйста, разрешите доступ к локальным файлам" },
-    "html_permissions_info": { "message": "У расширения в данный момент нет разрешения приостанавливать вкладки с URL (адресами), начинающимися со слова \"file\".\nЧтобы это изменить, включите \"Разрешить открывать локальные файлы по ссылкам\" со страницы настроек расширения." },
-    "html_permissions_backup": { "message": "Примечание: изменение этой опции приведет к временному исчезновению всех приостановленных вкладок.\nНа всякий случай рекомендуем сделать бэкап (резервную копию) текущей сессии вкладок." },
+    "html_permissions_info": { "message": "У расширения в данный момент нет разрешения приостанавливать вкладки с URL-адресами, начинающимися со слова \"file\".\nЧтобы это изменить, включите \"Разрешить открывать локальные файлы по ссылкам\" со страницы настроек расширения." },
+    "html_permissions_backup": { "message": "Примечание: изменение этой опции приведет к временному исчезновению всех приостановленных вкладок.\nНа всякий случай рекомендуем сделать бэкап (резервную копию) вкладок текущей сессии." },
     "html_update_title": { "message": "Обновление" },
     "html_update_heading2": { "message": "становится всё лучше" },
     "html_update_info": { "message": "Новая версия The Great Suspender готова к установке.\nПримечание: в процессе обновления приостановленные вкладки могут временно пропасть." },
-    "html_update_info_withbackup": { "message": "На всякий случай рекомендуем сделать бэкап (резервную копию) текущей сессии вкладок." },
-    "html_update_info_nobackup_prefix": { "message": "Рекомендуем сделать экспорт текущей сессии" },
+    "html_update_info_withbackup": { "message": "На всякий случай рекомендуется сделать резервную копию вкладок текущей сессии." },
+    "html_update_info_nobackup_prefix": { "message": "Рекомендуется сделать экспорт текущей сессии" },
     "html_update_info_nobackup_suffix": { "message": " перед обновлением расширения." },
     "html_updated_title": { "message": "Обновлено" },
     "html_updating_heading": { "message": "обновляется" },
     "html_updating_message": { "message": "Восстанавливаются приостановленные вкладки..." },
     "html_updated_great_suspender_updated": { "message": "успешно обновлено" },
-    "html_updated_info_line1": { "message": "Сделан автоматически бэкап перед обновлением." },
-    "html_updated_info_line2_prefix": { "message": "Если есть проблемы с обновлением, то Вы можете восстановить потерянные вкладки из бэкапа: " },
+    "html_updated_info_line1": { "message": "Перед обновлением была сделана резервная копия ваших вкладок." },
+    "html_updated_info_line2_prefix": { "message": "Если возникнут проблемы с обновлением, то Вы сможете восстановить потерянные вкладки из резервной копии: " },
```

Comment:
I do not know why there are these symbols here, in the original there is none. where can i see this message live? maybe you can give me a screenshot?
![image](https://user-images.githubusercontent.com/23438485/52112613-0e26b880-2618-11e9-97ab-92f44fd1d07e.png)

Reply:
This message I guess we can only see right before Chrome Web Store TGS version gets updated, so here @deanoemcke's help may also be needed

Reply:
![image](https://user-images.githubusercontent.com/1051763/52184513-00bc3900-2879-11e9-9cbb-833496ecbd4e.png)

Reply:
If you open the extension settings page, and change the url from options.html to updated.html you'll see this screen. 
The ...line2_prefix text is for the text before the 'session management' link.
The ...line2_suffix text is for after that link.

Reply:
Aww yes, this message appears AFTER the update, not before, so "html_updated_info_line2_prefix" might need to be like:
"Если возникли проблемы с обновлением, то Вы можете восстановить потерянные вкладки из резервной копии в разделе '"
and "html_updated_info_line2_suffix" like:
"'."
Soo that live it looks like:
Если возникли проблемы с обновлением, то Вы можете восстановить потерянные вкладки из резервной копии в разделе 'Управление сессиями'.
Where "Управление сессиями" is the "session management" link.
And I'm not sure whether "Управление сессиями" will start with the capital letter "У" or not ("у"). Coz if it's small "у" then I guess there's no need for ' around.
Also I would recommend to check translation that all russian "Вы", "Вас", "Ваш", etc, start with capital "В", so it all looks unified in showing user some "respect".

Reply:
@deanoemcke 
I get a slightly different message. for some reason, the messages are different, in version 160 there is text from version 109, which is no longer used?
![image](https://user-images.githubusercontent.com/23438485/52198714-d23c5f00-2874-11e9-8834-69002e9c1e16.png)

Reply:
@MonsterNya well messages r different probably coz a lot has changed since 109 version, but I can't get about which no longer used text r u talking here? 🤔

Reply:
@MonsterNya saw ur final variant, I think "html_updated_info_line2_prefix" at least should be like:
"Если во время обновления возникли проблемы, то Вы можете восстановить потерянные вкладки из резервной копии в разделе "
if not my previous variant then smth like that, coz it's not after, it's during the update (as it said in the original) user may encounter problems..

Reply:
@Jollfye 
well, if so in the original, let it be "во время". variant "с обновлением" was more suitable for tools to solve update problems than tab loss. 
fixed